Gopkg.lock

```diff
@@ -152,6 +152,14 @@
   revision = "ea4d1f681babbce9545c9c5f3d5194a789c89f5b"
   version = "v1.2.0"
 
+[[projects]]
+  digest = "1:797ad58a3838dd37d2e6fae09344f06ab927fa20974f0fd5198ece977246b44b"
+  name = "github.com/iov-one/tendermint"
```

Comment:
This is your problem.

Don't import this, but rather from tendermint/tendermint.
I'm sure this is where the types issue came from... I will mark it below

Reply:
Hmmm... guess you fixed the code, you can reset Gopkg.lock to master, to avoid the unneeded and misleading change.